sys/aarch64/signal.c

```diff
@@ -35,7 +35,6 @@ int sig_send(signo_t sig, sigset_t *mask, sigaction_t *sa, ksiginfo_t *ksi) {
   _REG(uctx, X0) = sig;
   _REG(uctx, X1) = sc_info;
   _REG(uctx, X2) = sc_uctx;
-  _REG(uctx, SP) -= sizeof(intptr_t *);
```

Comment:
Could you explain why we do not need it?

Reply:
We do it on MIPS because apparently the calling convention allows the callee to modify the top of the stack without allocating space itself. There is no such thing on ARM AFAIK.